FrontEnd/src/components/ProfileDetail/MainInfo/ProfileDetailNavBar.jsx

```diff
@@ -0,0 +1,48 @@
+import { HashLink } from 'react-router-hash-link';
+import classes from './ProfileDetailNavBar.module.css';
+
+function ProfileDetailNavBar() {
+  const menuItems = [
+    'Про компанію',
+    'Стартап',
+    'Товари / послуги',
+    'Логістика товарів / послуг',
+    'Формат співпраці',
+  ];
+  const menuLinks = [
+    'about-company',
+    'startup',
+    'products-services',
+    'logistics',
+    'cooperation',
+  ];
+
+  const activeHash = window.location.hash.substring(1);
```

Comment:
Could we use a `useParams` hook for it?

Reply:
There is no hash in the Route, so I didn't manage to use useParams hook. But I used useLocation instead.